src/components/inputs/time-input/time-input-body.styles.js

```diff
@@ -98,6 +98,7 @@ const getClockIconContainerStyles = (props, theme) => {
     border-bottom-right-radius: ${overwrittenVars[
       designTokens.borderRadiusForInput
     ]};
+    transition: ${overwrittenVars.transitionStandard};
```

Comment:
We must remember to update these in version 10.0.0, to specify the properties to animate.